README.md

```diff
@@ -347,6 +360,14 @@ Ignore breakpoints in files stored in `node_modules` folder or ending in `.test.
 ```
 $ node-debug --hidden node_modules/ --hidden \.test\.js$ app
 ```
+Only show files with the word 'test.js' in the path:
+```
+$ node-debug --hidden=^\(?\!.*test\.js.*\) app
+```
+Only show files with the word 'test.js' or 'hello.js' in the path:
+```
+$ node-debug --hidden=^\(?\!\(.*test\.js.*\)\|\(.*hello\.js.*\)\) app
+```
```

Comment:
I prefer to remove this change, because it is a verbose explanation of 
>Ignore breakpoints in files stored in `node_modules` folder or ending in `.test.js`:
 ```
 $ node-debug --hidden node_modules/ --hidden \.test\.js$ app
 ```

Reasonable to extend extend existed part to something like:

>Ignore breakpoints in files stored in `node_modules` folder or ending in `.test.js`
Hide this files in sources tree:
 ```
 $ node-debug --hidden node_modules/ --hidden \.test\.js$ app
 ```